### DISCUSSION TITLE
Self-nomination for Maintainer role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#146] [Ruby Team]
Hi @paketo-buildpacks/ruby-maintainers 

I would like to self-nominate for the role of maintainer for the Ruby team. I am actively interested in continuing to provide direction for the product and I am willing and able to share the responsibilities of maintainership.

Thanks,
Rob

cc @paketo-buildpacks/steering-committee

**Comment by ryanmoran** [comment]
Congrats on becoming a maintainer, @robdimsdale!
